src/XMonad/ManageHook.hs

```diff
@@ -91,7 +92,7 @@ className :: Query String
 className = ask >>= (\w -> liftX $ withDisplay $ \d -> fmap resClass $ io $ getClassHint d w)
 
 -- | A query that can return an arbitrary X property of type 'String',
---   identified by name.
+-- identified by name. Works for ASCII strings only.
```

Comment:
Did you perhaps wish to add a note that for `_NET_WM_NAME`, `WM_NAME` and possibly also `WM_CLASS` there are special functions available that work better, and hyperlink to them?

Reply:
Yes, quite right.

Reply:
I’ve just added a short sentence referring to the specialized variants. Is that fine? Any more suggestions?